public/app/features/scenes/components/VizPanel/VizPanelRenderer.tsx

```diff
@@ -14,14 +14,16 @@ import { SceneDragHandle } from '../SceneDragHandle';
 import { VizPanel } from './VizPanel';
 
 export function VizPanelRenderer({ model }: SceneComponentProps<VizPanel>) {
-  const { title, options, fieldConfig, pluginId, pluginLoadError, $data, ...state } = model.useState();
+  const { title, options, fieldConfig, pluginId, pluginLoadError, $data, layout: layoutOptions } = model.useState();
   const [ref, { width, height }] = useMeasure();
   const plugin = model.getPlugin();
   const { data } = sceneGraph.getData(model).useState();
-  const layout = sceneGraph.getLayout(model);
+  const parentLayout = sceneGraph.getLayout(model);
 
-  const isDraggable = layout.state.isDraggable ? state.isDraggable : false;
-  const dragHandle = <SceneDragHandle layoutKey={layout.state.key!} />;
+  // TODO: this should probably be parentLayout.isDraggingEnabled() ? layoutOptions?.isDraggable : false
+  // The current logic is not correct, just because parent layout itself is not draggable does not mean children are not
```

Comment:
When I added this logic I was thinking about a use cases where:
1. parentLayout.isDraggable controls drag and drop for all items, regardless of their own setting
2. parentLayout.isDraggable = true still allows some children not to be draggable, i.e. static/fixed elements.

Reply:
@dprokop  I get that, that is still there , I think? (see code below )

Reply:
Hmm maybe it should be parentLayout.state.isDraggable ? (So it's a property on the layout scene object not the child layout options prop)